tests/network/netwrok_integration_test.go

```diff
@@ -0,0 +1,107 @@
+// Copyright 2021 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package network
+
+import (
+	cfg "github.com/ChainSafe/gossamer/config"
+	"github.com/ChainSafe/gossamer/dot/rpc/modules"
+	"github.com/ChainSafe/gossamer/lib/common"
+	libutils "github.com/ChainSafe/gossamer/lib/utils"
+	"github.com/ChainSafe/gossamer/tests/utils"
+	"github.com/ChainSafe/gossamer/tests/utils/config"
+	"github.com/ChainSafe/gossamer/tests/utils/node"
+	"github.com/ChainSafe/gossamer/tests/utils/retry"
+	"github.com/ChainSafe/gossamer/tests/utils/rpc"
+	"github.com/adrg/xdg"
+	"github.com/stretchr/testify/require"
+
+	"context"
+	"os"
+	"testing"
+	"time"
+)
+
+func TestKadDHTNetworkDiscovery(t *testing.T) {
```

Comment:
This test actually does not work locally for me and it times out. Based on my logs it seems that Alice progresses in the network but Bob and Charlie do not

Reply:
hmmm. how do you run it? it works fine for me, trying to undersrtand what I missed

Reply:
retried and seems to be good now